views/searchWithFilter.njk

```diff
@@ -169,7 +169,7 @@
                             <div class="govuk-grid-column-one-half">
                               <span class="govuk-visually-hidden">Status</span>
                                 {{ govukTag({
-                                  text: u.status,
+                                  text: "Inactive",
```

Comment:
why have you changed that back?

Reply:
Merge issue, reverted to main branch changes